terraform/fec_rds.tf

```diff
@@ -73,10 +70,10 @@ resource "aws_db_instance" "rds_production" {
   }
   engine = "postgres"
   engine_version = "9.6.1"
```

Comment:
I think I called this out before, but we're definitely excited to use this as an opportunity to also upgrade to 9.6.1.  Thank you!